### PR TITLE
Fix: Typo in metric name

### PR DESCRIPTION
**Pull request by AleksandrKent**

### CODE REVIEW

File: python/usearch/index.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def __init__(
         :param metric: Distance function, defaults to MetricKind.IP
         :type metric: Union[MetricKind, Callable, str], optional
             Kind of the distance function, or the Numba `cfunc` JIT-compiled object.
-            Possible `MetricKind` values: IP, Cos, L2sq, Haversine, Pearson,
+            Possible `MetricKind` values: IP, Cos, L2_sq, Haversine, Pearson,
```
Comment (AleksandrKent) [inline python/usearch/index.py:202]: Maybe I mad a mistake to correct MetricKind values of L2sq to L2_sq at this line. I thought this should match `str` values for metrics.

Comment (ashvardanian) [reply python/usearch/index.py:202]: It’s either PascalCase or snake_case. Enums are generally the first, strings are the second in most Python repos.

Comment (AleksandrKent) [reply python/usearch/index.py:202]: @ashvardanian okay, there is still other typo in `_normalize_metric` in my commit above.

**Review by ashvardanian — APPROVED** [review]

**Pull request merged**